arch-gulpfile.js

```diff
@@ -242,6 +242,18 @@ const tasks = {
   "test-server-dev": () => shell.test("-d", "test/server") && exec(`mocha -c --opts ${config.mocha}/mocha.opts test/server`)
 };
 
+// Check if any .eslintrc file exists on the
+// project root path.
+const customLinterPath = Path.join(process.cwd(), ".eslintrc");
```

Comment:
Since `eslint --init` can generate files like `.eslintrc.json`, can this accept:

1. no extension
2. `.js`
3. `.json`?